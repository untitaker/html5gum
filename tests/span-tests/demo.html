this is a tag: <h1>test</h1>

tags can have attributes: <div id = foobar>

Attribute values can be quoted: <input name = 'age' type = "number">

But you cannot put attributes everywhere: </nope data=foobar>

Please mind the gap: < test

The pirate says &arrrrr;

Does this open two pages? <a href=foo.html href=bar.html>click me</a>

Do you start or do you end? </yes/>
