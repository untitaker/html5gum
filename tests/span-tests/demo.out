note:
  ┌─ test.html:1:1
  │
1 │ this is a tag: <h1>test</h1>
  │                 ^^       ^^ end tag
  │                 │
  │                 start tag
2 │
3 │ tags can have attributes: <div id = foobar>
  │                                ^^   ^^^^^^ attr value
  │                                │
  │                                attr name
4 │
5 │ Attribute values can be quoted: <input name = 'age' type = "number">
  │                                        ^^^^    ^^^  ^^^^    ^^^^^^ attr value
  │                                        │       │    │
  │                                        │       │    attr name
  │                                        │       attr value
  │                                        attr name
